srcpkgs/siril/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'siril'
+pkgname=siril
+version=0.9.12
+revision=1
+archs="x86_64"
```

Comment:
`archs` is only for cases where a software cannot possibly be used on another architecture. Is that the case with siril? I could imagine that it could at least run with the musl libc and possibly also on other CPU architectures.

Reply:
musl just needs libexecinfo:

```
case "$XBPS_TARGET_MACHINE" in
        *-musl) makedepends+=" libexecinfo-devel" ;;
esac
```

will fix that.

Reply:
Cross-compilation can be fixed by putting `gettext-devel` in hostmakedepends instead of makedepends.

Reply:
I put in the x86_64 restriction because it is the only arch I can currently test the package. If you consider it compiling good enough, I'm fine removing it.

Reply:
I tried cross compilation against a few targets. It compiles fine against armv7, aarch64 and i686.

 It uses the execinfo.h header for backtrace generation when catching signals so it won't compile for musl out of the box. Making it work would require either patching away the backtrace generation or finding a workaround.

Reply:
I applied the linkage against libexecinfo for the musl build and it built fine. I've added the change and removed the arch restriction, although I've only actually tested the software on an amd64 glibc system.